### DISCUSSION TITLE
Unable to import geopandas from geopandas-0.9.0-py2.py3-none-any.whl /geopandas-0.8.2-py2.py3-none-any.whl in AWS glue

### DISCUSSION THREAD

**Discussion by a3di** [geopandas/geopandas#2743] [Installation]
-
---

#### System information

AWS Glue

#### Environment details

<details>

Failed to get options via gdal-config: [Errno 2] No such file or directory: 'gdal-config': 'gdal-config'
    A GDAL API version must be specified. Provide a path to gdal-config using a GDAL_CONFIG environment variable or use a GDAL_VERSION environment variable.

Command '['/usr/local/bin/python', '-m', 'pip', 'install', '--target=/glue/lib/installation', '/tmp/glue-python-libs-c07gfrbt/geopandas-0.7.0-py2.py3-none-any.whl']' returned non-zero exit status 1.
</details>

**Comment by jorisvandenbossche** [comment]
I suppose this is a failure in installing `fiona`, one of the dependencies of geopandas. And based on the error, it seems it was trying to install it from source, and not using a binary wheel. 

What OS / platform is being used in your AWS Glue environment?
